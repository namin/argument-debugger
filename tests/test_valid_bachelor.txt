All bachelors are unmarried men.
John is a bachelor.
Therefore, John is an unmarried man.