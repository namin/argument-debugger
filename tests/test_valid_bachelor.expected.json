{
  "description": "Valid deductive argument - bachelor definition",
  "must_find": [],
  "may_find": [],
  "must_not_find": ["circular", "false_dichotomy", "missing_link", "unsupported_premise"]
}